### ISSUE TITLE
Week 3 Todo List

### ISSUE THREAD

**Issue by rogercmq** [rogercmq/NetworkCompressionPaperNotes#5]
Phase 1:

- [ ] pytorch ImageNet-pretrained VGG
- [ ] pytorch VGG trained from stretch on CIFAR100

Phase 2:
- [ ] Finetune both networks on sub-dataset (e.g. 15 classes on CIFAR100)
- [ ] Network Slimming them and also finetune.

Phase 3:
- [ ] Test the accuracy of model A (finetuned on sub-dataset A) on sub-dataset B;
- [ ] Test the accuracy of model B (finetuned on sub-dataset B) on sub-dataset A;
